pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-8388_drug-list-update.xml

```diff
@@ -0,0 +1,586 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="DDP-8388_druglist" author="LKapanad">
+        <insert tableName="drug_list">
+            <column name="generic_name" type="varchar(200)" value="AMIVANTAMAB"/>
```

Comment:
as far as I know we don't need to add the type here. would make the whole thing better readable